src/lib/deliveryManager.ts

```diff
@@ -30,11 +22,25 @@ export type DeliverToDestinationListInput = {
   destinationFilename?: string;
 };
 
+export type DeliveryFnForDestinationTypeInput = {
```

Comment:
`DeliverToDestinationInput`, `DeliverToDestinationListInput` & `DeliveryFnForDestinationTypeInput` type names are all very similar and a little confusing - I wonder if splitting `deliverToDestination` and `deliverToDestinations` in separate files would help? Or maybe renaming `deliverToDestinations` to `processDeliveries` - and then the types could be renamed to match the function names like `ProcessDeliveriesInput` ?

Reply:
Yep, good call... I was not crazy about the naming either. I like `ProcessDeliveries` as a replacement.